spec/support/shared_examples/persistence_strategy.rb

```diff
@@ -0,0 +1,36 @@
+shared_examples 'a persistence strategy' do
+  shared_context 'with changes' do
+    before do
+      subject.obj << RDF::Statement.new(RDF::Node.new, RDF::DC.title, 'moomin')
+    end
+  end
+
+  describe '#persist!' do
+    xit 'evaluates false with no changes' do
```

Comment:
Ah, `@persisted` doesn't set to false if the in memory graph changes, so it won't serve here.  Going back to my original proposal.

I guess this is what happens when you set down half-finished work for a few months.